crates/rolldown/tests/rolldown/topics/bundler_esm_cjs_tests/27/artifacts.snap

```diff
@@ -17,7 +17,7 @@ var import_foo = __toESM(require_foo());
 
 //#endregion
 //#region entry.js
-input.works = import_foo.default[Math.random() < 1 && "default"].bar === 123 && import_foo.bar === void 0;
+input.works = import_foo.default[Math.random() < 1 && "default"].bar === 123 && import_foo.default.bar === void 0;
```

Comment:
This is expected, since https://github.com/rolldown/rolldown/blob/e4bd3ef32fdb51faa8f9c0480e03678b5522d345/crates/rolldown/tests/rolldown/topics/bundler_esm_cjs_tests/27/foo.js?plain=1#L1-L2 is neither `cjsReexport` nor a cjs module with static `exports`